### PR TITLE
Harshit/voice cloning

### PR DESCRIPTION
**Pull request by harshitag1**
In this pull request I have implemented voice cloning using Play.ht instant voice cloning API. I have written the required functionality both on the server and the client side. The demo page is also made and the documentation of both server and client side is also done.

### Files Changed
packages/usellm/src/client/types.ts
packages/usellm/src/client/usellm.ts
packages/usellm/src/server/llm-service.ts
packages/usellm/src/server/types.ts
packages/usellm/src/shared/utils.ts
usellm.org/app/api/llm/route.ts
usellm.org/app/demo/clone-voice/page.tsx
usellm.org/config/demos.ts
usellm.org/config/docs.ts
usellm.org/content/docs/api-reference/cloneVoice.mdx
usellm.org/content/docs/api-reference/llmservice-cloneVoice.mdx

### CODE REVIEW

File: usellm.org/app/demo/clone-voice/page.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+export default function CloneVoice() {
+  const [status, setStatus] = useState<Status>("idle");
+  const [audioUrl, setAudioUrl] = useState<string>("");
+  const [text, setText] = useState<string>("");
+  const [voiceID, setVoiceID] = useState<string>("");
+  const [audioUrlReturn, setAudioUrlReturn] = useState<string>("");
+  const [name, setName] = useState<string>("");
+  const [clonedVoiceID, setClonedVoiceId] = useState<string>("");
+  const llm = useLLM({
+    serviceUrl: "/api/llm", // For testing only. Follow this guide to create your own service URL: https://usellm.org/docs/api-reference/create-llm-service
```
Comment (BirajCoder) [inline usellm.org/app/demo/clone-voice/page.tsx:16]: Change the service url

File: usellm.org/app/demo/clone-voice/page.tsx
Comment (BirajCoder) [inline usellm.org/app/demo/clone-voice/page.tsx:1]: Add docs

Comment (harshit-jovian) [reply usellm.org/app/demo/clone-voice/page.tsx:1]: Done

File: usellm.org/app/demo/clone-voice/page.tsx (hunk 1 above)
Comment (harshit-jovian) [reply usellm.org/app/demo/clone-voice/page.tsx:16]: Done

**Review by BirajCoder — APPROVED** [review]

**Pull request merged**